<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditAdrenalineNode</class>
 <widget class="QDialog" name="AddEditAdrenalineNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Piratecash Node</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255); color: rgb(0,0,0); border: 1px solid #c4c1bd;</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="text">
        <string>Alias*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="aliasLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="text">
        <string>Address*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="text">
        <string>PrivKey*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="privkeyLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="text">
        <string>TxHash*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txhashLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="text">
        <string>Output Index*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputindexLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="text">
        <string>Reward Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rewardaddressLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="text">
        <string>Reward %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rewardpercentageLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(74, 195, 195);
color: rgb(0, 0, 0);
border-radius: 4px;
border-color:  rgb(74, 195, 195);
font-weight: 600;
padding: 4px;
padding-left: 10px;
padding-right: 12px;
icon-size: 0;</string>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border-radius: 4px;
border: 1px solid #c4c1bd;
font-weight: 400;
padding: 4px;
padding-left: 10px;
padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
